pkg/Microsoft.DotNet.ILCompiler/Microsoft.DotNet.ILCompiler.pkgproj

```diff
@@ -5,44 +5,31 @@
     <BaseLinePackageDependencies>false</BaseLinePackageDependencies>
     <PackagePlatforms>x64;</PackagePlatforms>
     <PreventImplementationReference>true</PreventImplementationReference>
+    <RestorePackages>true</RestorePackages>
     <SkipValidatePackage>true</SkipValidatePackage>
     <IncludeSymbolsInPackage Condition="'$(IncludeSymbolsInPackage)' == ''">true</IncludeSymbolsInPackage>
-
+    <!-- Mark this package as a lineup (meta-) package -->
+    <IsLineupPackage Condition="'$(IsLineupPackage)' == ''">true</IsLineupPackage>
     <!-- Override this property so that the package name won't look like runtime.[RID].[TFM].[ID] -->
     <PackageTargetRuntime></PackageTargetRuntime>
   </PropertyGroup>
+  
   <ItemGroup>
-    <File Include="$(MSBuildThisFileDirectory)\Microsoft.DotNet.ILCompiler.targets" >
+    <File Include="$(PackageSourceDirectory)\BuildIntegration\Microsoft.DotNet.ILCompiler.targets">
       <TargetPath>build</TargetPath>
     </File>
-    <File Include="$(PackageSourceDirectory)\BuildIntegration\*">
+    
+    <File Include="$(PackageSourceDirectory)\BuildIntegration\*" Exclude="$(PackageSourceDirectory)\BuildIntegration\Microsoft.DotNet.ILCompiler.targets">
```

Comment:
Is there a reason why we have the one target in the build subdirectory and other in the targets subdirectory? Can we just have all of them in the build subdirectory? 

From what I can tell, other nuget packages seems to be authored that way - they just have the targets in the build directory.

Reply:
(If there is a good reason, I would be nice to add comment about it.)

Reply:
Not really - I wanted to make a distinction between targets that are imported into the project at restore-time versus others, which are consumed at build-time. 
Moved.